tests/nested_pandas/nestedframe/test_nestedframe.py

```diff
@@ -320,6 +320,43 @@ def test_recover_from_flat():
     assert nf2.equals(nf)
 
 
+def test_from_lists():
+    """Test NestedFrame.from_lists behavior"""
+    nf = NestedFrame(
+        {"c": [1, 2, 3], "d": [2, 4, 6], "e": [[1, 2, 3], [4, 5, 6], [7, 8, 9]]}, index=[0, 1, 2]
+    )
+
+    # Test a few combinations
+    res = NestedFrame.from_lists(nf, base_columns=["c", "d"], name="nested_e")
+    assert list(res.columns) == ["c", "d", "nested_e"]
+    assert list(res.nested_columns) == ["nested_e"]
+
+    res = NestedFrame.from_lists(nf, base_columns=["c", "d"], list_columns=["e"])
```

Comment:
Can we add a test case where we specify multiple columns in `list_columns`?

Reply:
Added coverage for this!